src/rust/ide/src/view/node_editor.rs

```diff
@@ -208,6 +213,8 @@ impl GraphEditorIntegratedWithController {
 
         let node_removed = Self::ui_action(&model,
             GraphEditorIntegratedWithControllerModel::node_removed_in_ui,&invalidate.trigger);
+        let nodes_collapsed = Self::ui_action(&model,
```

Comment:
can we align those `=` please?

Reply:
We don't align if they are not in subsequent lines. I even have memories of asking once for the very same change.